server.go

```diff
@@ -10,6 +10,8 @@ import (
 	"os"
```

Comment:
Some time ago I started separating the code into packages. At the beginning, everything was in the main package (root). Since then, I moved quite a lot of code to packages like `proxy`, `actions` and so on. I started moving the server.go code into the `server` package but did not get far (there's always something more urgent to do). Maybe those new functions would be a good candidates to put them directly into `server/server.go`.

This is not something really important so please feel free to ignore this comment altogether.